core/controllers/reader_test.py

```diff
@@ -1189,6 +1189,221 @@ def test_remove_collection_from_incomplete_list_handler(self):
                 self.user_id), [])
 
 
+class StorePlaythroughHandlerTest(test_utils.GenericTestBase):
+    """Tests for the handler that records playthroughs."""
+
+    def setUp(self):
+        super(StorePlaythroughHandlerTest, self).setUp()
+        self.exp_id = '15'
+
+        self.login(self.VIEWER_EMAIL)
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        exp_services.delete_demo(self.exp_id)
+        exp_services.load_demo(self.exp_id)
+        self.exploration = exp_services.get_exploration_by_id(self.exp_id)
+        playthrough_id = stats_models.PlaythroughModel.create(
+            self.exp_id, self.exploration.version, 'EarlyQuit',
+            {
+                'state_name': {
+                    'value': 'state_name1'
+                },
+                'time_spent_in_exp_in_msecs': {
+                    'value': 200
+                }
+            },
+            [{
+                'action_id': 'ExplorationStart',
+                'action_customization_args': {
+                    'state_name': {
+                        'value': 'state_name1'
+                    }
+                }
+            }], True)
```

Comment:
nit: use keyword argument here so it's clearer "what" is going to be set to `True`.

Reply:
Done.